Test/Entities/DogEntity.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
```

Comment:
there are no failure tests. what happens if you instantiate `DogEntity` with the incorrect datatypes?

Reply:
(liliyia)

Reply:
done